jupyter-notebooks/analytics/user-guide/05_summary_statistics_ships_and_clouds.ipynb

```diff
@@ -0,0 +1,434 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 7. Normalize and clean the report data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The graph above is likely very noisy due to clouds, haze, and a variation in the amount of imagery per day. The steps below normalize the object count by the estimated area of usable imagery that the model observed. Planet currently provides two versions of an unusable data mask (UDM) for most scenes. Udm (version 1) is less accurate but is available for every scene. Udm2 is more accurate but is sometimes unavailable. The steps below use udm2 to estimate the percentage of pixels that are usable (i.e. not cloudy), and the original udm to estimate the total imaged area per day."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pd.set_option('precision', 15)\n",
+    "\n",
+    "# Get the total area of the subscription or submitted feature (sq m)\n",
+    "submitted_area = df['Submitted Area'][0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### a) Remove time points that contain < 50% clear imagery\n",
+    "On cloudy days results are less likely to be accurate."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df['Clear Percentage'] = df['Clear Area (udm2_band_1)'] / df['Total Area (udm2)']\n",
+    "df = df[df['Clear Percentage'] > 0.5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### b) Remove time points where imagery coverage is < 50%\n",
+    "If only a small section of the AOI contains imagery, inferring the object count for the whole AOI is less accurate."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df['Imagery Coverage'] = df['Total Area (udm2)'] / submitted_area\n",
+    "df = df[df['Imagery Coverage'] > 0.5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### c) Estimate usable area per time point\n",
+    "Models can often detect objects through light haze and sometimes through heavy haze, so we use that rough information to create an estimated \"usable percentage\" metric. You can adjust the parameters if you know the model your using performs better or worse in haze."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a column that estimates the percentage of imagery where the model is expected to perform.\n",
+    "df['Usable Percentage'] = (df['Clear Area (udm2_band_1)'] + df['Light Haze Area (udm2_band_4)'] + (df['Heavy Haze Area (udm2_band_5)'] / 2)) / df['Total Area (udm2)']\n",
```

Comment:
You could make the `/2` a configurable variable, e.g. 
```
LIGHT_HAZE_FRAC = 1.0
HEAVY_HAZE_FRAC = 0.5
df['Usable Percentage'] = (df['Clear Area (udm2_band_1)'] + df['Light Haze Area (udm2_band_4)'] * LIGHT_HAZE_FRAC + (df['Heavy Haze Area (udm2_band_5)'] * HEAVY_HAZE_FRAC)) / df['Total Area (udm2)']
```

Reply:
Great idea!